app/assets/javascripts/admin/addon/templates/site-text-edit.hbs

```diff
@@ -13,6 +13,11 @@
     @class="site-text-value"
   />
 
+  {{#if this.siteText.has_interpolation_keys}}
+    <div class="desc">Available interpolation keys:
+      {{this.interpolationKeys}}</div>
```

Comment:
Nitpick but would  put the </div> next line :p

Reply:
Me, too. Will try, but this is what Prettier wanted me to do. 😅

Reply:
Oh I see I missed the text above.

So two things:
- we need to have a local for: "Available interpolation keys:"
- and then you can most likely do `<div class="desc">{{i18n "foo"  keys=this.interpolationKeys}}</div>`